ui/legacy/src/components/cstudio-admin/mods/content-type-propsheet/contentTypes.js

```diff
@@ -23,7 +23,7 @@
   const STAR = '*';
 
   function createLookupTableFromCVS(cvs) {
-    return (cvs || '')
+    return !cvs ? [] : cvs
```

Comment:
Looks like it was correct before this change: you are expecting a string; not an array.

Reply:
no because it is returning an empty array of  "", "".slice is == [""]